tests/system/libraries/ChromeLib.py

```diff
@@ -67,7 +67,10 @@ def close_chrome_tab(self):
 			f"{process.is_process_running(ChromeLib._processRFHandleForStart)}"
 		)
 
-		if not windowsLib.isWindowInForeground(ChromeLib._chromeWindow):
+		if (
+			not ChromeLib._chromeWindow

```

Comment:
If this if condition is True due to _chromeWindow being None, won't the log message fail as it won't be able to fetch `.title` etc?
Perhaps handle _chromeWindow in its own if condition first somehow?

Reply:
right yes, makes sense